spec-0005/index.md

```diff
@@ -25,6 +26,35 @@ intended use-cases, and pseudo-code illustrating its use.
 Discuss how this would be implemented.
 -->
 
+### Build nightly wheels
+
+By building and uploading wheel for your project you make it easier for projects
+that depend on you to test against your latest changes. They can then report problems
+they find.
+
+There are a few steps to implementing this for your project:
+1. Get access to the area that nightly wheels are uploaded to
+2. Setup a CI step that builds wheels for your project
+3. Setup a CI step that uploads wheels to https://anaconda.org/scientific-python-nightly-wheels/
+
+For step (1) contact XXX. Deposit the token you get as a [secret on your repository](https://docs.github.com/en/actions/security-guides/encrypted-secrets) named `UPLOAD_TOKEN`.
+
+The work for step (2) depends on your project. You are probably already doing this for your
+releases. The new thing to add is that building wheels is run on a schedule every night or
+once a week.
+
+For step (3) there is a GitHub Action that you can use. You can find the action at
+https://github.com/scientific-python/upload-nightly-action. To use it in your "build wheels
```

Comment:
This looks pretty barebones. Any interest to adopt or collaborate with https://github.com/OpenAstronomy/github-actions-workflows ?

Reply:
Something for @Carreau maybe who created the GitHub action

Reply:
Sure, we can look into that as well. No particular objections. Those action seem pretty complex ?